### PR TITLE
Difference of Squares: Added params to address Issue #555 referenced …

### LINKED ISSUE #555: difference-of-squares: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by atg-abhishek**
…in Issue #509

Fixes #555

### CODE REVIEW

File: exercises/difference-of-squares/difference_of_squares.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-def square_of_sum():
+def square_of_sum(number_of_numbers):
```
Comment (m-a-ge) [inline exercises/difference-of-squares/difference_of_squares.py]: This doesn't sound natural. Maybe a better option can be found?

Comment (atg-abhishek) [reply exercises/difference-of-squares/difference_of_squares.py]: Only for this function or for all the functions? Perhaps just `N` would suffice now that I think about it, given the reference in the README.md?

Comment (m-a-ge) [reply exercises/difference-of-squares/difference_of_squares.py]: for all of them of course 😄 
I don't like `N`  even it is stated in `README.md` that way, it develops a bad habit of naming your variables\parameter names to ambiguous things and your code suffers from it. Do you see my point?

A good Python style guide from Google - https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

**Comment by N-Parsons** [comment]
@atg-abhishek, are you still working on this?

**Comment by N-Parsons** [comment]
I'm going to go ahead and close this due to inactivity. If you want to continue it, feel free to reopen.

**Comment by atg-abhishek** [comment]
Hi! Sorry unfortunately I wasn't able to work on it!

**Pull request closed without merging**